app/controllers/KeycloakConfigurationController.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import auth.OidcSecurity
+import javax.inject.Inject
+import org.pac4j.play.scala.SecurityComponents
+import play.api.Configuration
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, Request}
+
+class KeycloakConfigurationController @Inject ()(val controllerComponents: SecurityComponents,
+                                                 configuration: Configuration
+                                                ) extends OidcSecurity {
+
+  def keycloak: Action[AnyContent] = secureAction { implicit request: Request[AnyContent] =>
+    val authUrl = configuration.get[String]("auth.url")
+    Ok(Json.obj(
+      "auth-server-url" -> s"$authUrl/auth",
+      "resource" -> "tdr-fe",
```

Comment:
Is this the client ID? I'm just trying to work out how this all fits together. 🙂 

Presumably you have to set up a new non-confidential client in Keycloak? Could you add some instructions to the readme about how to do that?

Also, do we have a card for automating the local Keycloak setup? Could you add one to the backlog if not?

Reply:
It is the client ID. I've added a line to the README about setting up another client and I've added a card to the backlog to add the exported keycloak config to our private configuration repository. For the time being, local setup will include donwloading and importing that file although when we have time, we can probably automate it into the docker image fully.